twitcher/datatype.py

```diff
@@ -50,7 +50,10 @@ def has_purl(self):
     def public(self):
         """Flag if service has public access."""
         # TODO: public access can be set via auth parameter.
-        return self.get('public') or False
+        value = self.get('public', False)
+        if value is True or value == 1 or value == 'true':
```

Comment:
Maybe use `pyramid.settings.asbool` which handles more use cases, but this is still ok for now.